app/views/layouts/_common_ngapp.html.haml

```diff
@@ -0,0 +1,37 @@
+= render :partial => 'layouts/doctype'
```

Comment:
Right.. but given that is the only change, can't we just do a conditional ng-app in `application`? Feels like an overkill to copy a whole layout just to add one attribute..